src/App.js

```diff
@@ -12,27 +12,30 @@ import ProtectedRoute from './components/ProtectedRoute';
 
 function App() {
   const [isLoggedIn, setIsLoggedIn] = useState(false);
+  const cookies = new Cookies();
 
   useEffect(() => {
-    setIsLoggedIn(false);
-    if (Cookies.get('token')) {
-      console.log('token: ' + Cookies.get('token'));
+    // setIsLoggedIn(false);
+    if (cookies.get('token')) {
+      console.time();
       setIsLoggedIn(true);
-      // Cookies.remove('token');
+      console.timeEnd();
```

Comment:
I assume these were for debugging? 
Generally, you want to get rid of your console logs before committing (though this is easy to forget, which is where linting and code review can help!)